includes/wc-admin-update-functions.php

```diff
@@ -316,3 +316,42 @@ function wc_admin_update_290_update_apperance_task_option() {
 function wc_admin_update_290_db_version() {
 	Installer::update_db_version( '2.9.0' );
 }
+
+/**
+ * Use woocommerce_admin_activity_panel_inbox_last_read from the user meta to set wc_admin_notes.is_read col.
+ */
+function wc_admin_update_300_update_is_read_from_last_read() {
+	global $wpdb;
+	$user      = wp_get_current_user();
+	$last_read = get_user_meta( $user->ID, 'woocommerce_admin_activity_panel_inbox_last_read', true );
```

Comment:
If this is run in the background, I wonder who the current user is?  The one who initially updated or the one who most recently viewed any admin page?

Minor, but obviously this could lead to setting odd "read" statuses depending on who the current user is.

Reply:
That's a really good point. I honestly did not test it at all.

I think we should use the latest `woocommerce_admin_activity_panel_inbox_last_read` value.

Reply:
@joshuatf I updated code to use the latest `woocommerce_admin_activity_panel_inbox_last_read` value in https://github.com/woocommerce/woocommerce-admin/pull/7945/commits/ba012f63b0814e1877fc2a33a0924c3f2fcbd9a8

Reply:
The updated code to find the latest `woocommerce_admin_activity_panel_inbox_last_read` value looks good to me 👍